packages/realm-server/tests/module-syntax-test.ts

```diff
@@ -86,7 +86,7 @@ module(basename(__filename), function () {
         @field firstName = contains(StringField);
         @field age = contains(NumberField);
         static embedded = class Embedded extends Component<typeof this> {
-          <template><h1><@fields.firstName/></h1></template>                
+          <template><h1><@fields.firstName/></h1></template>
```

Comment:
The tests were previously assertion that we **add** whitespace here, which was a bug.